srcpkgs/geonkick/template

```diff
@@ -9,9 +9,10 @@ makedepends="jack-devel libsndfile-devel rapidjson
 short_desc="Free software percussion synthesizer"
 maintainer="teldra <teldra@rotce.de>"
 license="GPL-3.0-or-later"
-homepage="https://github.com/iurie-sw/geonkick"
-distfiles="https://github.com/iurie-sw/geonkick/archive/v${version}.tar.gz"
-checksum=ec656e76df13ae1226bba6eeb8146c4f0ddbae2c34d28a89db473c16497eaff4
+homepage="https://github.com/free-sm/geonkick"
+changelog="https://github.com/free-sm/geonkick/blob/main/NEWS.md"
+distfiles="https://github.com/free-sm/geonkick/archive/refs/tags/v${version}.tar.gz"
+checksum=813fb10709136ed79403c8baf3d847fce9fbf717fea3c194ad2952e0172e6d45
 make_check=extended # No target to "ninja test".
 
 if [[ ! ${XBPS_TARGET_MACHINE} == "x86_64" ]]; then
```

Comment:
```suggestion
if ! [ ${XBPS_TARGET_MACHINE} = "x86_64" ]; then
```

but on aarch64 systems, it will default to aarch64, so this doesn't really make sense for native compilation, so
a) change this to run only when cross compiling and trust them to detect the architecture correctly
b) set it in configure_args every the time